app/app.py

```diff
@@ -0,0 +1,26 @@
+"""Main App module"""
+
+from flask import Flask, render_template, url_for
```

Comment:
Remove unused imports

Reply:
I ended up using them as shown in this commit `d57b997`
But I shall take note not to include unused imports.
Thanks.